stacks/signal-processing/nifi.yaml

```diff
@@ -5,10 +5,12 @@ metadata:
   name: nifi
 spec:
   image:
-    productVersion: 1.27.0
+    productVersion: 1.28.1
     # TODO (@NickLarsenNZ): Use a versioned image with stackable0.0.0-dev or stackableXX.X.X so that
     # the demo is reproducable for the release and it will be automatically replaced for the release branch.
-    custom: oci.stackable.tech/stackable/nifi:1.27.0-postgresql
+    # custom: oci.stackable.tech/stackable/nifi:2.2.0-postgresql
+    custom: oci.stackable.tech/stackable/nifi:1.28.1-postgresql
+    # pullPolicy: IfNotPresent
```

Comment:
Did you mean to check this in?

Reply:
The commented out pullPolicy? Yes. It kept bugging me since one of the demos needs a locally built image (until changes are merged to main and it is pushed to the image registry).

Reply:
Works for me ;)